pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -1144,10 +1165,10 @@ struct LowerTileToPXAPass : public LowerTileToPXABase<LowerTileToPXAPass> {
                                 ResultIs<EltwiseInteger>>,
         ContractionOpConversion<CombinationKind::eq,
                                 CmpFloatOp<CmpFPredicate::OEQ>,
-                                ResultIs<EltwiseFloat>>,
+                                AnyComparandIs<EltwiseFloat>>,
```

Comment:
I probably just don't understand some magic MLIR goo, but how is AnyComparandIs related to ComparandsAre?

Reply:
Oh wait, I get it.  Basically float happens on *either* being float (which already exists), and int happens only when both are int....